stable/kibana/templates/configmap.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "kibana.fullname" . }}
+  labels:
+    app: {{ template "kibana.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+data:
+  config: |
+    # Default Kibana configuration from kibana-docker.
+    server.name: kibana
+    server.host: "0"
```

Comment:
What do these options do? If replicas > 1 are these settings safe?

Reply:
These setting are safe. They are the default kibana cluster name and the host name/ip to bind to. These are provided as defaults from the kibana-docker image and do not affect HA.